### ISSUE TITLE
Remind feature?

### ISSUE THREAD

**Issue by MuawizChaudhary** [mmohades/Venmo#13]
Hi, will the ability to remind people of a payment be implemented in this API?

**Comment by mmohades** [comment]
Hey,
How'd you do it on the Venmo app itself?

**Comment by MuawizChaudhary** [comment]
Theres a button to do it when you tap on "Triple Bars" then "Incomplete". There are two buttons; one to remind, one to cancel the request. There are also options to pay out a payment request in another tab, that you can find if you look at the "payments" tab.

**Comment by mmohades** [comment]
Thanks for opening an issue. I've released version 0.2.0 with the changes.
You'll need to get payments first, and then send a reminder for a specific payment
```python
payments = client.payment.get_charge_payments()
client.payment.remind_payment(payments[0])

# you can also cancel a payment
client.payment.cancel_payment(payments[0])

# or use a payment_id
client.payment.remind_payment(payment_id=payments[0].id
client.payment.cancel_payment(payment_id=payments[0].id)
```
Let me know if you find any issues with it.
 Run `pip3 install venmo-api --upgrade` to update to the new version.
